src/imitation/data/rollout.py

```diff
@@ -397,25 +408,22 @@ def generate_trajectories(
         Sequence of trajectories, satisfying `sample_until`. Additional trajectories
         may be collected to avoid biasing process towards short episodes; the user
         should truncate if required.
-
```

Comment:
Why are we removing this?

Reply:
Because gym.spaces.Dict doesn't have shape and I don't find this check helpful or provide valuable information for later code. Maybe I missed something here, given we support Dict obs now, do you still think it is necessary to check here?

![image](https://github.com/HumanCompatibleAI/imitation/assets/38305983/0d06f2f4-7b29-4568-9178-3ee4f73a9873)

Reply:
OK, makes sense!